content/installation/from-packages.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: bt_wiki
-title: Installation Using Packages for Common OS Distributions
+title: Installing Cloudify from the Command Line Interface
```

Comment:
This is for installing Cloudfiy Command Line Interface (Cloudify CLI tool) ie we should drop the word "from"